From cbe5031a10bbe6658a08d6f4bffd6ff35a3889fa Mon Sep 17 00:00:00 2001
From: Svet Ganov <svetoslavganov@google.com>
Date: Thu, 15 Dec 2016 14:51:17 -0800
Subject: [PATCH] [DO NOT MERGE] Fix vulnerability in MemoryIntArray - fix
 build file

bug:33039926
bug:33042690

Change-Id: If0431b77ec546c72f8cc25bb605a851572bb22a6
(cherry picked from commit c3db570a0064b2dcbe806ddb5de3f678623612ca)
---
 core/tests/utiltests/Android.mk | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/core/tests/utiltests/Android.mk b/core/tests/utiltests/Android.mk
index d4a6493..4013c4c 100644
--- a/core/tests/utiltests/Android.mk
+++ b/core/tests/utiltests/Android.mk
@@ -25,3 +25,5 @@ LOCAL_PACKAGE_NAME := FrameworksUtilTests
 LOCAL_CERTIFICATE := platform
 
 include $(BUILD_PACKAGE)
+
+include $(call all-makefiles-under,$(LOCAL_PATH))
